Esri/arcgis-maps-sdk-swift-samples
# [New] Add Custom Dynamic Entity Data Source

## Description

This PR implements `Add custom dynamic entity data source` in `Layers` category.
URL to README: 

## Linked Issue(s)

- `swift/issues/4383`

## How To Test
- Dynamic entities should start to appear on the map
- Tap on a dynamic entity and all the attributes for that entity should appear in a callout

## Screenshots
![add-custom-dynamic-entity-data-source](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/74069582/9a3a712b-188b-4549-abc6-3528d6184552)
